spec/src/main/resources/META-INF/smithy/bsp/extensions/java.smithy

```diff
@@ -50,6 +50,7 @@ structure JavacOptionsItem {
     /// the -classpath flag in the command line interface
     /// of javac.
     @required
+    @deprecated(message: "This might be empty, use the buildTarget/jvmCompileClasspath endpoint")
```

Comment:
```suggestion
    @deprecated(message: "When the jvmCompileClasspath capability is supported, this might be empty and the buildTarget/jvmCompileClasspath endpoint should be used instead.")
```

Or anything else, that makes clear, when this can be empty.